benchmarks/Microsoft.Extensions.ObjectPool.Performance/ObjectPoolBenchmark.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Running;
+
+namespace Microsoft.Extensions.ObjectPool
+{
+    public class ObjectPoolBenchmark
```

Comment:
Maybe name it `DefaultObjectPoolBenchmark`? So further ObjectPool-benchmarks can be added and distinguished.

Reply:
The `DefaultObjectPool` has a shortcut for the default policy, so `Return` is not addressed by the benchmark. A benchmark with [StringBuilderPooledObjectPolicy](https://github.com/gfoidl/Benchmarks/blob/1af788adb4b444a81b071f8673edf45196c72d1d/aspnet/Common/ObjectPool/ObjectPool/Benchmarks/StringBuilderFullPoolBenchmarks.cs) addressed this. I can send an PR after this one is merged.

Reply:
OK cool. Not doing the rename because someone can always just rename it in the future if necessary. YAGNI